lib/CustomFields/components/FieldAccordion/edit-fields/SingleSelectDropdownFields.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Row,
+  Col,
+  ConfirmationModal,
+} from '@folio/stripes-components';
+
+import {
+  HiddenField,
+  NameField,
+  HelpTextField,
+  RequiredField,
+  OptionsFields,
+} from './shared-fields';
+import AddOptionButton from './components';
+
+const propTypes = {
+  deleteOption: PropTypes.func.isRequired,
+  fieldNamePrefix: PropTypes.string.isRequired,
+  values: PropTypes.arrayOf(PropTypes.string),
+};
+
+const defaultProps = {
+  values: {},
+};
+
+const SingleSelectDropdown = ({ fieldNamePrefix, values, deleteOption }) => {
+  const [data, setData] = useState(values?.selectField?.options?.values || [{}]);
```

Comment:
I would suggest using FieldArray in OptionsFields.js to avoid having this state.